src/Rector/Contrib/Symfony/Controller/RedirectToRouteRector.php

```diff
@@ -74,10 +81,13 @@ public function refactor(Node $node): ?Node
     }
 
     /**
+     * @param MethodCall $node
+     *
      * @return mixed[]
      */
     private function resolveArguments(Node $node): array
```

Comment:
I think here it could be typehint instead of `Node`, right?

Reply:
Right, thanks for the catch, gonna search for similars :+1: